src/template/point-template.js

```diff
@@ -0,0 +1,49 @@
+import { formatStringToTime, FormatStringToShortDate, getPointDuration } from '../utils.js';
+
+export function createPointTemplite(point, pointDestination, pointOffers){
+  const { basePrice, type, isFavorite, dateFrom, dateTo} = point;
+  const { name} = pointDestination;
+  return `<li class="trip-events__item">
+  <div class="event">
+    <time class="event__date" datetime=${dateFrom}>${FormatStringToShortDate(dateFrom)}</time>
+    <div class="event__type">
+      <img class="event__type-icon" width="42" height="42" src="img/icons/${type}.png" alt="Event type icon">
+    </div>
+    <h3 class="event__title">${type} ${name}</h3>
+    <div class="event__schedule">
+      <p class="event__time">
+        <time class="event__start-time" datetime=${dateFrom}>${formatStringToTime(dateFrom)}</time>
+        &mdash;
+        <time class="event__end-time" datetime=${dateTo}>${formatStringToTime(dateTo)}</time>
+      </p>
+      <p class="event__duration">${getPointDuration(dateFrom,dateTo)}</p>
+    </div>
+    <p class="event__price">
+      &euro;&nbsp;<span class="event__price-value">${basePrice}</span>
+    </p>
+
+    <h4 class="visually-hidden">Offers:</h4>
+      <ul class="event__selected-offers">
+      ${pointOffers.map((e)=> (
+    `<li class="event__offer">
```

Comment:
лучше сразу такое сокращение убери, а то есть критери Б4,
```suggestion
      ${pointOffers.map((evt)=> (
    `<li class="event__offer">
```